src/targets/gpu/kernels/include/migraphx/kernels/index.hpp

```diff
@@ -155,7 +155,7 @@ struct index
         return max_nlocal() / nlocal_subwave<SubWaveSize>();
     }
 
-    constexpr index_constant<__AMDGCN_WAVEFRONT_SIZE> nlocal_wave() const { return {}; }
+    constexpr index_constant<MIGRAPHX_WAVEFRONTSIZE> nlocal_wave() const { return {}; }
```

Comment:
The tidy error is from the `kernel_file_check`, you need to add a the define there(you can just set it to `64` for now): 

https://github.com/ROCm/AMDMIGraphX/blob/f56b1b4f14bfa198ad4c17befb7c35592fbae7ef/src/targets/gpu/CMakeLists.txt#L120